doc/man1/openssl-list.pod

```diff
@@ -64,6 +65,12 @@ B<bar>.
 In verbose mode, the algorithms provided by a provider will get additional
 information on what parameters each implementation supports.
 
+=item B<-kdf-algorithms>
+
+Display a list of key derivation function algorithms.
+If a line is of the form C<foo =E<gt> bar> then B<foo> is an alias for the
+official algorithm name, B<bar>.
```

Comment:
You can remove that sentence, as that only happens with legacy implementations

Reply:
(ah, I just noticed that can be done with macs too. Will do)

Reply:
I've done both.